client/components/banner/index.jsx

```diff
@@ -48,7 +48,8 @@ export class Banner extends Component {
 		event: PropTypes.string,
 		feature: PropTypes.string,
 		href: PropTypes.string,
-		icon: PropTypes.string,
+		icon: PropTypes.oneOfType( [ PropTypes.string, PropTypes.bool ] ),
```

Comment:
Why are we updating the types here?

Reply:
Oops, that should have been changed back in the last commit. Good catch!